app/assets/javascripts/host_checkbox.js.erb

```diff
@@ -40,13 +40,17 @@ function readFromCookie() {
 }
 
 function toggle_actions() {
-  var dropdown = $("#submit_multiple a");
+  var dropDownContainer = $("#submit_multiple");
+  var dropdown = dropDownContainer.find("a");
+  var disabledMessage = __("Please select hosts to perform action on.");
```

Comment:
would it be easier to add the title when creating the action in the first place? e.g. at https://github.com/theforeman/foreman/blob/develop/app/helpers/hosts_helper.rb#L175 ?